src/Tasks/Error.cs

```diff
@@ -31,6 +31,11 @@ public sealed class Error : TaskExtension
         /// </summary>
         public string HelpKeyword { get; set; }
 
+        /// <summary>
+        /// A link pointing to more information about the error
+        /// </summary>
+        public string HelpLink { get; set; }
```

Comment:
Shouldn't these be passed through in the call to `Log.LogError`?

Reply:
Yes, I already thought that I missed something somewhere. Fixed that now :).